README.md

```diff
@@ -3,13 +3,7 @@
 [![GitHub Build Status](https://github.com/cisagov/ansible-role-admiral/workflows/build/badge.svg)](https://github.com/cisagov/ansible-role-admiral/actions)
 [![CodeQL](https://github.com/cisagov/ansible-role-admiral/workflows/CodeQL/badge.svg)](https://github.com/cisagov/ansible-role-admiral/actions/workflows/codeql-analysis.yml)
 
-This is a skeleton project that can be used to quickly get a new
-[cisagov](https://github.com/cisagov) Ansible role GitHub project
-started.  This skeleton project contains
-[licensing information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for an Ansible role.
+Ansible Role that installs [cisagov/admiral](https://github.com/cisagov/admiral).
```

Comment:
This should be rephrased because it is installing a Docker composition to run the Admiral and not the Admiral software itself.

Reply:
Rephrased in [c8bce9c](https://github.com/cisagov/ansible-role-admiral/pull/3/commits/c8bce9c2484df31220eaf1217e7c74ef92428020).

Reply:
It looks like you rebased and that commit is _gone_ but in https://github.com/cisagov/ansible-role-admiral/pull/3/commits/c7d9e2061fd1ee8b6f0f91ec224f6c966b0a947b it appears to be correctly doing what I asked.